### PR TITLE
Add actionCartGetPackageShippingCost hook

### PR DESCRIPTION
**Pull request by PrestaEdit**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Allow to edit the package shipping cost value.
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Add a module with actionCartGetPackageShippingCost  hook and update shippingCost value.

### CODE REVIEW

File: classes/Cart.php [hunk 1]
```diff
@@ -3548,6 +3548,53 @@ public function getPackageShippingCost(
...
+     * @param bool $use_tax
+     * @param Country|null $default_country
+     * @param array|null $product_list list of product concerned by the shipping.
+     *                                 If null, all the product of the cart are used to calculate the shipping cost
+     * @param int|null $id_zone Zone ID
+     * @param bool $keepOrderPrices When true use the Order saved prices instead of the most recent ones from catalog (if Order exists)
+     *
+     * @return float|bool Shipping total, false if not possible to ship with the given carrier
+     */
+    public function getPackageShippingCostValue(
```
Comment (Progi1984) [inline classes/Cart.php]:
```suggestion
    protected function getPackageShippingCostValue(
```

**Review by Progi1984 — DISMISSED** [review]

**Comment by Hlavtox** [comment]
Good!!!

Needed this yesterday to add extra charge to shipping, had to use actionDeliveryPriceByPrice and recalculate the price again… 😬

**Comment by Hlavtox** [comment]
Good!!!

Needed this yesterday to add extra charge to shipping, had to use actionDeliveryPriceByPrice and recalculate the price again… 😬

**Comment by atomiix** [comment]
Thanks @PrestaEdit! Can you resolve the conflicted file?

**Review by matks — CHANGES_REQUESTED** [review]
@PrestaEdit I'm sorry :( you performed a `merge` not a `rebase` so now your branch contains a merge commit and is refused by CI tool Pull Request Validator

**Pull request closed without merging**